content/en/_index.md

```diff
@@ -216,6 +276,11 @@ public static void exampleSnippet(String projectId, String filePath) {
     // Snippet content ...
 }
 {{< /tab >}}
+{{< tab header="Python" >}}
+# This is an example snippet for showing best practices.
+def example_snipet(project_id: str, file_path: str):
```

Comment:
Should we call out optional types here like you did above?

Reply:
Which optional types?